cubeviz/layout.py

```diff
@@ -224,10 +240,10 @@ def mouse_move(self, event):
                             ra, dec = wcs.wcs_pix2world([[x, y]], 0)[0]
 
                         if ra is not None and dec is not None:
-                            string = string + " " + self._format_coord_string(ra, dec)
+                            string = string + " " + self._coords_format_function(ra, dec)
                 # Pixel Value:
-                v = arr[x][y]
-                string += " {:1.4f}".format(v)
+                v = arr[y][x]
```

Comment:
X and Y need to be flipped when indexing.

Reply:
For future it would be good to put a few lines of comments as to *why* it is supposed to be flipped when indexing.  That way we, including future-@robelgeda , won't have to think too hard to figure it out again.